build/get_cd_deps.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+echo "BUILD_BRANCH: ${BUILD_BRANCH}"
+if [[ $BUILD_BRANCH == "staging" ]]; then
+  echo "Building staging"
+  npm install --save cp-translations@staging @coderdojo/cd-common@staging
+elif [[ $BUILD_BRANCH =~ ^pull\/[0-9]+$ ]]; then
+  echo "Building PR"
+  PR_ID=`echo ${BUILD_BRANCH} | cut -d '/' -f 2`
+  PR_JSON=`curl "https://api.github.com/repos/coderdojo/cp-zen-frontend/pulls/${PR_ID}" 2>/dev/null`
+  PR_BRANCH_NAME=`echo $PR_JSON | jq -r '.head.ref'`
+  PR_TARGET_BRANCH_NAME=`echo $PR_JSON | jq -r '.base.ref'`
+  echo "Source branch: ${PR_BRANCH_NAME}"
+  echo "Target branch: ${PR_TARGET_BRANCH_NAME}"
+
+  # cp-translations
+  CP_TRANSLATIONS_BRANCH_STATUS=`curl -I "https://api.github.com/repos/${GITHUB_USERNAME}/cp-translations/branches/${PR_BRANCH_NAME}" 2>/dev/null | head -n 1 | cut -d$' ' -f2`
```

Comment:
yeah. backticks way of being run by bash is worst then `$()`. `$()` will be more reliable and would give the ability to nest commands